net/sshuttle/files/sshuttle.init

```diff
@@ -0,0 +1,111 @@
+#!/bin/sh /etc/rc.common
+#
+# Copyright (C) 2022 Thomas Cujé
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+# shellcheck disable=SC2034
+START=99
+STOP=01
+USE_PROCD=1
+
+_log() {
+  logger -p daemon.info -t sshuttle "$@"
+}
+
+_err() {
+	logger -p daemon.err -t sshuttle "$@"
+}
```

Comment:
This is specific for sshuttle or not? Because you should check how other packages are doing it.

Reply:
This is not sshuttle specific, there are at least 7 other init files that use this function and I copied from there